charts/external-es-proxy/templates/_helpers.tpl

```diff
@@ -65,12 +65,12 @@ Create the name of the service account to use
 {{- define "external-es-proxy-trustcerts" -}}
 {{- if .Values.global.customLogging.trustCaCerts  }}
 {{- $secret_name := .Values.global.customLogging.trustCaCerts }}
-  proxy_ssl_trusted_certificate /etc/ssl/certs/{{ $secret_name }}.pem;
-  proxy_ssl_verify              on;
-  proxy_ssl_verify_depth        2;
-  proxy_ssl_session_reuse on;
+proxy_ssl_trusted_certificate /etc/ssl/certs/{{ $secret_name }}.pem;
```

Comment:
Adding comments for easier review

Eg
# Specify the trusted SSL certificate for proxy connections
`proxy_ssl_trusted_certificate /etc/ssl/certs/{{ $secret_name }}.pem;`